imports/ui/statistics/PlayerModal.js

```diff
@@ -0,0 +1,28 @@
+import Players from '../../api/Players';
+import Constants from '../../api/Constants';
+
+import './PlayerModal.html';
+
+Template.PlayerModal.helpers({
+	getPlayerInfo() {
+		let player = Session.get("selectedStatistics");
+		if (player) {
+			let info = {};
+			info.name = player.leagueName;
+			info.elo = player.elo.toFixed(3);
+			info.winRate = (player.handsWin / player.handsTotal * 100).toFixed(1);
```

Comment:
At the very least, I think average hand size and average position don't have enough information with only one decimal place, especially position.  Thoughts?